### ISSUE TITLE
[a11y]: error state for slider input is failing Use of Color as well as Error Identification

### ISSUE THREAD

**Issue by mbgower** [carbon-design-system/carbon#9772]
### Package

carbon-components-react

### Browser

Chrome

### Operating System

MacOS

### Package version

unknown. Just looking at the react storybook demo

### Automated testing tool and ruleset

manual inspection

### Assistive technology

n/a issue is with presentation; 

### Description

issue described on https://ibm-studios.slack.com/archives/G01GCBCGTPV/p1632933032011300

### WCAG 2.1 Violation

1.4.1 Use of Colour; 3.3.1 Error Identification

### CodeSandbox example

https://react.carbondesignsystem.com/?path=/story/components-slider--default

### Steps to reproduce

1. navigate to input in slider
2. change value to a number outside the range of the slider
3. Tab away from the input. The dotted outline around the input changes from blue to red, but no other indication is given of the input being in error, nor is a message displayed to the user.

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by tay1orjones** [comment]
Duplicate of https://github.com/carbon-design-system/carbon/issues/10153

**Comment by tay1orjones** [comment]
Going to close this one in favor of the one above since there's some more conversation around potential solutions to the problem in that issue.
